upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -205,55 +143,50 @@ There you can find links to upgrade notes for other versions too.
-    - add `shopsys/frontend-api` dependency with `composer require shopsys/frontend-api`
-    - register necessary bundles in `config/bundles.php`
-        ```diff
-            Shopsys\FormTypesBundle\ShopsysFormTypesBundle::class => ['all' => true],
-        +   Shopsys\FrontendApiBundle\ShopsysFrontendApiBundle::class => ['all' => true],
-        +   Overblog\GraphQLBundle\OverblogGraphQLBundle::class => ['all' => true],
-        +   Overblog\GraphiQLBundle\OverblogGraphiQLBundle::class => ['dev' => true],
-            Shopsys\GoogleCloudBundle\ShopsysGoogleCloudBundle::class => ['all' => true],
-        ```
-    - add new route file [`config/routes/frontend-api.yaml`](https://github.com/shopsys/shopsys/blob/master/project-base/config/routes/frontend-api.yaml) from GitHub
-    - add new route file [`config/routes/dev/frontend-api-graphiql.yaml`](https://github.com/shopsys/shopsys/blob/master/project-base/config/routes/dev/frontend-api-graphiql.yaml) from GitHub
-    - copy [type definitions from Github](https://github.com/shopsys/shopsys/tree/master/project-base/config/graphql/types) into `config/graphql/types/` folder
-    - copy necessary configuration [shopsys_frontend_api.yaml from Github](https://github.com/shopsys/shopsys/blob/master/project-base/config/packages/shopsys_frontend_api.yaml) to `config/packages/shopsys_frontend_api.yaml`
+    - run these steps only in case you have not recently updated to Symfony Flex as is described at beginning of this file
+        - add `shopsys/frontend-api` dependency with `composer require shopsys/frontend-api`
+        - register necessary bundles in `config/bundles.php`
+            ```diff
+                Shopsys\FormTypesBundle\ShopsysFormTypesBundle::class => ['all' => true],
+            +   Shopsys\FrontendApiBundle\ShopsysFrontendApiBundle::class => ['all' => true],
+            +   Overblog\GraphQLBundle\OverblogGraphQLBundle::class => ['all' => true],
+            +   Overblog\GraphiQLBundle\OverblogGraphiQLBundle::class => ['dev' => true],
+                Shopsys\GoogleCloudBundle\ShopsysGoogleCloudBundle::class => ['all' => true],
+            ```
+        - add new route file [`config/routes/frontend-api.yaml`](https://github.com/shopsys/shopsys/blob/master/project-base/config/routes/frontend-api.yaml) from GitHub
+        - add new route file [`config/routes/dev/frontend-api-graphiql.yaml`](https://github.com/shopsys/shopsys/blob/master/project-base/config/routes/dev/frontend-api-graphiql.yaml) from GitHub
+        - copy [type definitions from Github](https://github.com/shopsys/shopsys/tree/master/project-base/config/graphql/types) into `config/graphql/types/` folder
```

Comment:
Is there any info about updating `services.yaml` as well ? 
```
    resolverMaps:
        namespace: App\
        resource: '../src/**/*{Map}.php'
        tags:
            - { name: overblog_graphql.resolver_map, schema: default }
```
I can't find such an info

Reply:
This is part of frontend api bundle, so users do not need to do any of this